File: go/02-Counting-the-Number-of-Characters/count.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print("What is the input string? ")
	myString, _ := reader.ReadString('\n')
	myString = strings.Replace(myString, "\n", "", -1)

	myCount := len(myString)

	output := myString + " has " + strconv.Itoa(myCount) + " characters."

	fmt.Println(output)

}

File: python/02-counting-the-number-of-characters/count.py
#!/usr/bin/env python3
"""
Prompt for an input string and print the number of characters
"""


def main():
    """
    Prompt for an input string and print the number of characters
    """
    my_string = input("What is the input string? ")
    num_of_chars = len(my_string)
    print(my_string + " has " + str(num_of_chars) + " characters.")


if __name__ == '__main__':
    main()
